### DISCUSSION TITLE
toc-debug 是？AntD Table 组件文档页的一个丢失目录奇怪问题

### DISCUSSION THREAD

**Discussion by esmadrider** [ant-design/ant-design#42836] [Q&A] [answered]
我之前从外部搜过一个如何拖拽列宽度的文章，发现了 https://ant.design/components/table#components-table-demo-resizable-column 这个不存在的目录。此时目录是这样的：

<img width="124" alt="image" src="https://github.com/ant-design/ant-design/assets/19285461/a66e6210-6ccd-47f5-97ec-a65afdbad995">

我之后再次从 https://ant.design/components/overview 中直接搜索 Table 进去后路由是 https://ant.design/components/table，目录是这样的：

<img width="144" alt="image" src="https://github.com/ant-design/ant-design/assets/19285461/46761c72-341b-4b09-b17f-27734a887a77">

导致我再也搜不到 resizable 的目录了，排查了半天才发现是以上的复现逻辑。有紫色的目录属于隐藏的目录？这几个紫色的隐藏目录都有一个 class 为 toc-debug 的 span 标签，什么情况呀？

源码只搜到了 CSS https://github.com/ant-design/ant-design/blob/master/.dumi/theme/slots/Content/index.tsx#L62，不知道哪里使用了这个 CSS，为什么出现这种目录？请教！

**Comment by MadCcc** [comment]
是 debug 用的 demo
![image](https://github.com/ant-design/ant-design/assets/27722486/d60bbc7f-8806-4339-8957-5e9dc5c0a7f1)

**Reply by esmadrider** [reply]
> 是 debug 用的 demo ![image](https://user-images.githubusercontent.com/27722486/243324218-d60bbc7f-8806-4339-8957-5e9dc5c0a7f1.png)

好的，感谢，回复的好快。

那么再请问下~，“调试专用演示”主要解决什么问题呢，我觉得我这个目录挺好的，为什么被隐藏啦 🤔

**Reply by MadCcc** [reply, marked as answer]
正常 Demo 展示的都是推荐用法，debug 用的 demo 大部分都是方便线上 debug 的，同时也会用来做截图测试
